{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "b6930ad4_185a4bd9",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1181219
      },
      "writtenOn": "2021-06-05T23:10:06Z",
      "side": 1,
      "message": "Another Python 3 fix. PTAL.",
      "revId": "ad9d3c6b3f9d4f82675e035214ecfccda2ff53f6",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "4b483191_86763636",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1373330
      },
      "writtenOn": "2021-06-06T06:32:23Z",
      "side": 1,
      "message": "LGTM.",
      "revId": "ad9d3c6b3f9d4f82675e035214ecfccda2ff53f6",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}